doc/debugging.md

```diff
@@ -99,3 +99,34 @@ Also to inspect logs in a more flexible manner, instead of the strict output fro
 adb logcat | grep 'ReactNativeJS\|StatusModule\|GoLog'
 ```
 
+### From @clauxx (visual debugging):
+
+When building/fixing/refactoring components, it can be annoying to set-up the component for instant visual feedback (i.e. finding it in the UI when fixing/refactoring; finding a place for it when building from scratch). Very often if the location of the screen containing your component nested deeply in the app, you might have to do a lot of repeated navigation, nevermind logging in every time the app reloads (even with fast refresh disabled this happens quite often).
+
+To alleviate this discomfort and make the development cycle more *pleasant*, you can use one of the following options (*_both of which involve the ... REPL ༼ つ ╹ ╹ ༽つ, so make sure you have that set-up with your IDE/editor_*)
+
+#### REPL-ing to a screen
+
+This one is straightforward, just evaluate a navigation dispatch form from anywhere:
+```cljs
+(comment (rf/dispatch [:navigate-to :your-favorite-buggy-screen]))
+```
+> TIP: you can run this from an untracked user namespace `(ns user)`, from which you can experiment or interact with the REPL. This is where I usually keep my development navigation forms.
```

Comment:
@clauxx I think we should always add empty lines between code sections and the rest of the text. For example, if I try to call `fill-paragraph` in Emacs to break this `> TIP ...` long line to be 80 chars wide, it doesn't work. In Vim `gqq` does though 🤷🏼 It also helps to navigate across paragraphs.

I kept postponing auto-formatting markdown in our repo, would love if somebody did this :P